pymc/distributions/timeseries.py

```diff
@@ -247,142 +289,298 @@ def logp(
         )
 
 
-class AR1(distribution.Continuous):
-    """
-    Autoregressive process with 1 lag.
+class AutoRegressiveRV(OpFromGraph):
+    """A placeholder used to specify a log-likelihood for an AR sub-graph."""
 
-    Parameters
-    ----------
-    k: tensor
-       effect of lagged value on current value
-    tau_e: tensor
-       precision for innovations
-    """
+    default_output = 1
+    ar_order: int
+    constant_term: bool
 
-    def __init__(self, k, tau_e, *args, **kwargs):
+    def __init__(self, *args, ar_order, constant_term, **kwargs):
```

Comment:
Can we add a docstring?

Reply:
What would it say? This init method clearly saves two variables as class instance properties and sends the rest to the super class.

Reply:
Ah nothing , I see I just missed it above. Thanks for pushing back